app/src/main/java/com/example/hw5/second/SwipeToDelete.kt

```diff
@@ -0,0 +1,23 @@
+package com.example.hw5.second
+
+import androidx.recyclerview.widget.ItemTouchHelper
+import androidx.recyclerview.widget.RecyclerView
+import com.example.hw5.model.Repository.singers
+import com.example.hw5.second.recycler.SingerListAdapter
+
+class SwipeToDelete (
+    var adapter: SingerListAdapter
```

Comment:
better provide lambda here
`onAction: ()->Unit`
and call it inside of onSwiped